src/pages/HomePage.js

```diff
@@ -0,0 +1,89 @@
+import {
+  ImageList, ImageListItem,
+  ImageListItemBar, Card, CardMedia,
+} from '@mui/material';
+import React, { useEffect } from 'react';
+import ArrowCircleDownRoundedIcon from '@mui/icons-material/ArrowCircleDownRounded';
+import FavoriteIcon from '@mui/icons-material/Favorite';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import fetchImages from '../api/data_fetch';
+import '../components/image/image_card.css';
+import NavgationBar from '../components/NavgationBar';
+import Loading from '../components/Loading';
+
+export default function HomePage() {
```

Comment:
- [ ] Your homepage should contain a filtering mechanism like a search box with which users can filter the list of images on the homepage. Kindly implement that.

  ![furkan-home](https://user-images.githubusercontent.com/36542986/163186705-bc0613ab-8c6f-4ab0-86d5-e0352f545d2b.jpg)

- [ ] For a better design I would recommend you fix the appearance of your list on the homepage in larger screen sizes.

  ![akumu-desktop](https://user-images.githubusercontent.com/36542986/163195126-20c49035-91be-4ca8-8750-ecdb3bc29cfb.jpg)